config/locales/en/data_exports.en.yml

```diff
@@ -1,6 +1,14 @@
 en:
+  data_export:
+    attachments: "Attachments"
+    file: "File %{n}"
+    status: "Export Status"
+    task: "Associated Task"
   data_exports:
     default_name: "%{type} on %{current_time}"
+    index_heading: "Reports"
```

Comment:
Can we call these data exports? I want to be consistent, and I think "reports" sets expectations about more sophisticated functionality than we're providing.